### PR TITLE
Add upak loader by name

### PR DESCRIPTION
**Pull request by mlsteele**

### CODE REVIEW

File: go/teams/create.go [hunk 1]
```diff
@@ -29,29 +29,17 @@ func CreateImplicitTeam(ctx context.Context, g *libkb.GlobalContext, impTeam key
...
-		}
-		return &upak.Current, nil
-	}
-
 	loadUsernameList := func(usernames []string) (res []*keybase1.UserPlusKeysV2, err error) {
 		for _, username := range usernames {
-			upak, err := loadUpakByUsername(username)
+			upak, _, err := g.GetUPAKLoader().LoadV2(libkb.LoadUserArg{
+				Name:       username,
+				NetContext: ctx,
```
Comment (oconnor663) [inline go/teams/create.go]: Sucks that it's so easy to forget to pass along this extra param.

File: go/teams/create.go [hunk 2]
```diff
@@ -29,29 +29,17 @@ func CreateImplicitTeam(ctx context.Context, g *libkb.GlobalContext, impTeam key
...
-			return nil, err
-		}
-		return &upak.Current, nil
-	}
-
 	loadUsernameList := func(usernames []string) (res []*keybase1.UserPlusKeysV2, err error) {
 		for _, username := range usernames {
-			upak, err := loadUpakByUsername(username)
+			upak, _, err := g.GetUPAKLoader().LoadV2(libkb.LoadUserArg{
+				Name:       username,
```
Comment (mlsteele) [inline go/teams/create.go]: cc @mmaxim now you can do this

Comment (mmaxim) [inline go/teams/create.go]: I thought the big fix here was to load by UID?

Comment (mmaxim) [inline go/teams/create.go]: I thought the big fix here was to load by UID?

Comment (mlsteele) [inline go/teams/create.go]: I'm not sure what you mean. This is CORE-5873 "give UPAK loader ability to load via username", and I assumed by the timing when you made that ticket that this piece of code was when you decided that that would be a nice feature.

**Pull request merged**